go.mod

```diff
@@ -1,3 +1,29 @@
-module github.com/networkservicemesh/cmd-template
+module github.com/networkservicemesh/cmd-nsc
 
 go 1.14
+
+require (
+	github.com/OneOfOne/xxhash v1.2.8 // indirect
+	github.com/antonfisher/nested-logrus-formatter v1.1.0
+	github.com/golang/protobuf v1.4.2
+	github.com/kelseyhightower/envconfig v1.4.0
+	github.com/networkservicemesh/api v0.0.0-20200804023023-3a49928a16bf
+	github.com/networkservicemesh/sdk v0.0.0-20200811232015-95542c51c8d9
+	github.com/open-policy-agent/opa v0.22.0 // indirect
+	github.com/opentracing/opentracing-go v1.2.0
+	github.com/pkg/errors v0.9.1
+	github.com/rcrowley/go-metrics v0.0.0-20200313005456-10cdbea86bc0 // indirect
+	github.com/sirupsen/logrus v1.6.0
+	github.com/spiffe/go-spiffe/v2 v2.0.0-alpha.5
+	github.com/stretchr/testify v1.6.1
+	github.com/uber/jaeger-client-go v2.25.0+incompatible // indirect
+	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de // indirect
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
+	golang.org/x/sys v0.0.0-20200810151505-1b9f1253b3ed // indirect
+	golang.org/x/text v0.3.3 // indirect
```

Comment:
I feel a few of these indirect can be removed.

Reply:
Fixed.